src/operators/elementAt.ts

```diff
@@ -0,0 +1,58 @@
+import Operator from '../Operator';
+import Observer from '../Observer';
+import Subscriber from '../Subscriber';
+import ArgumentOutOfRangeError from '../util/ArgumentOutOfRangeError';
+
+export default function elementAt(idx: number, defaultValue?: any) {
+  return this.lift(new ElementAtOperator(idx, defaultValue));
+}
+
+class ElementAtOperator<T, R> implements Operator<T,R> {
+  
+  idx: number;
+  defaultValue: any;
+  
+  constructor(idx: number, defaultValue?: any) {
+    if (idx < 0) {
+      throw new ArgumentOutOfRangeError;
+    }
+    
+    this.idx = idx;
+    this.defaultValue = defaultValue;
+  }
+  
+  call(subscriber: Subscriber<T>): Subscriber<T> {
+    return new ElementAtSubscriber(subscriber, this.idx, this.defaultValue);
+  }
+}
+
+class ElementAtSubscriber<T, R> extends Subscriber<T> {
+  
+  idx: number;
```

Comment:
please name `index` and make this `private`

`private index:number = 0;`

It's a good idea to use `private` or `protected` where appropriate, because we plan on using a TypeScript clojure-compiler-esque minifier from our friends at Google (@jeffbcross) that will use that information to rename properties and reduce output size.